.github/workflows/release.yml

```
@@ -0,0 +1,336 @@
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+*'
+
+env:
+  GO_VERSION: '1.24.5'
+
+jobs:
+  test:
+    name: Test Before Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)